php/login.php

```diff
@@ -0,0 +1,141 @@
+<?php
+require_once("database.php");
+
+session_start();
+if (isset($_SESSION["loggedin"]) && $_SESSION["loggedin"] === true) {
+  //needs to be point to main page after login
+  header("location: welcome.php");
+  exit;
+}
+
+$conn = connect_local();
+$email = $password = "";
+$email_err = $password_err = $login_err = "";
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+  if (empty(trim($_POST["email"]))) {
+    $email_err = "Please enter your email!";
+  } else {
+    $email = trim($_POST["email"]);
+  }
+  if (empty(trim($_POST["password"]))) {
+    $password_err = "Please enter your password!";
+  } else {
+    $password = trim($_POST["password"]);
+  }
+
+  if (empty($email_err) && empty($password_err)) {
+    $sql = "SELECT id, email, password FROM user WHERE email= ?";
+    $stmt = $conn->prepare($sql);
+    $param_email = trim($_POST["email"]);
+    $stmt->bindParam(1, $param_email);
+    if ($stmt->execute()) {
+      if ($stmt->rowCount() == 1) {
+        if ($row = $stmt->fetch()) {
+          $id = $row["id"];
+          $email = $row["email"];
+          //password is not yet hashed in the db
+          $hashed_password = $row["password"];
+          #if(password_verify($password, $hashed_password)){
+          if ($password === $hashed_password) {
+            session_start();
+            $_SESSION["loggedin"] = true;
+            $_SESSION["id"] = $id;
+            $_SESSION["email"] = $email;
+            //needs to be point to main page after login
+            header("location: welcome.php");
+          } else {
+            $login_err = "Invalid email or password.";
+          }
+        }
+      } else {
+        $login_err = "Invalid email or password.";
+      }
+    } else {
+      echo "Something went wrong. Please try again later.";
+    }
+    unset($stmt);
+  }
+  unset($conn);
+}
+?>
+
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+  <link rel="stylesheet" href="../css/login.css">
+  <script src="../js/login.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+  <title>Login</title>
+</head>
+
+<body>
+  <script>
```

Comment:
Please no script inside html files. 
We can have a look how to show/hide elements depending on a condition.